Polishing documentation, adding links to upstream resources (#33)
SeequentEvo/evo-data-converters: src/evo/data_converters/README.md

```

> **Note:** this example only returns the `ObjectMetadata`, with no option to not publish immediately.  Refer to the existing converter examples to see usage for the optional return of `BaseSpatialDataProperties_V1_0_1`.

Author: What do you think about "and will publish immediately"? I assume the intention here is to indicate that this behaviour can't be disabled or skipped, not sure if it's necessary to specify that explicitly.

> **Note:** this example only returns the `ObjectMetadata`, with no option to not publish immediately.  Refer to the existing converter examples to see usage for the optional return of `BaseSpatialDataProperties_V1_0_1`.

Reviewer: Yep that sounds good.

> **Note:** this example only returns the `ObjectMetadata`, with no option to not publish immediately.  Refer to the existing converter examples to see usage for the optional return of `BaseSpatialDataProperties_V1_0_1`.

Author: ```suggestion
**Note:** this example only returns the `ObjectMetadata`, and will publish immediately.  Refer to the existing converter examples to see usage for the optional return of `BaseSpatialDataProperties_V1_0_1`.
```
